test/e2e/operator/deployment_api.go

```diff
@@ -36,13 +37,17 @@ var (
 	scheme = runtime.NewScheme()
 )
 
-var _ = DescribeForAll("pmem-csi", func(o *Operator, f *framework.Framework) {
+var _ = deploy.DescribeForSome("API", func(d *deploy.Deployment) bool {
+	// Run these tests for all bare operator deployments, i.e.
+	// those which did not already install the driver.
+	return d.HasOperator && !d.HasDriver
+}, func(d *deploy.Deployment) {
+
+	f := framework.NewDefaultFramework("operator")
```

Comment:
We can reuse the framework initialized for deployment I guess.

Reply:
The deployment framework doesn't have a namespace created. We therefore need a new one here.